packages/abi-gen/package.json

```diff
@@ -38,7 +38,7 @@
         "generate_contract_wrappers": "run-p gen_typescript gen_python",
         "gen_typescript": "abi-gen --abis  ${npm_package_config_abis} --template ../../node_modules/@0x/abi-gen-templates/contract.handlebars --partials '../../node_modules/@0x/abi-gen-templates/partials/**/*.handlebars' --output ./test-cli/output/typescript --backend ethers",
         "gen_python": "pip install black && abi-gen --abis  ${npm_package_config_abis} --template ../../node_modules/@0x/abi-gen-templates/Python/contract.handlebars --partials '../../node_modules/@0x/abi-gen-templates/Python/partials/**/*.handlebars' --output ./test-cli/output/python --language Python",
-        "diff_contract_wrappers": "test-cli/diff.sh ./test-cli/expected-output ./test-cli/output",
+        "diff_contract_wrappers": "cd test-cli/output; git diff --exit-code .",
```

Comment:
does this change the current directory after running `yarn diff_contract_wrappers`? That seems undesirable. Would it be possible to do `git diff --exit-code test-cli/output` instead?

Reply:
I'm pretty positive that it doesn't make any difference, because I think every invocation of `yarn whatever` starts with its own shell process spawned with the current working directory set to the package directory, and any `cd`ing that happens in that script run is confined to that script run.

But, I really like the suggestion, so I changed it anyways :grin:

Reply:
ya you're totally right! It's fine on CI. `yarn test` runs `yarn diff_contract_wrappers` tho so could be confusing for devs.